### ISSUE TITLE
Reciva Services not online any more.... what now?

### ISSUE THREAD

**Issue by 4920441** [philsmd/sharpfin#57]
After a couple of years I found my trusty old shaprfin flashed internet radio again...

But the Reciva Servers seem to be down now....  I was kinda shocked that even my local URLs I added via the sharpfin patch don't work any more..

Does anyone else use such a nearly 16 year old relict, or am I the only one left to do so? :-)

Cheers

4920441

**Comment by EastEriq** [comment]
There is a lot you can still do, check https://iradioforum.net/forum/index.php?board=45.0
If you can still manage the urls of your presets I'd first check each one of them to make sure they still point to something streamable with the radio codecs, and if not search for new urls on some repository like https://www.radio-browser.info/ or https://fmstream.org/

**Comment by 4920441** [comment]
You are right.... the presets still work fine, though the screen nags about not finding any servers any more. I'll take a shot! :-)

[Edit] The most annoying problem is, that http**S** streams doesn't work (any more? Don't remember if they worked in the past, or simply modern ciphers are not supported any more, or a couple of CAs are too old on the device)  
Maybe Traefik or something could come to the rescue....

**Comment by EastEriq** [comment]
> The most annoying problem is, that http**S** streams doesn't work

IIRC http://www.megapico.co.uk/sharpfin/mediaserver.html supports them.

https://iradioforum.net/forum/index.php?topic=2969.0
